### ISSUE TITLE
Internal error while creating updates

### ISSUE THREAD

**Issue by talon256** [jprouty/mint-amazon-tagger#113]
Getting the following (new) error with 1.51 this morning. I can't recall if this is similar to the error I was getting just prior to the selenium error that 1.51 fixed that was waiting on the chromedriver update two weeks from then. If so, I suppose I'm still waiting on that.

Please let me know if I can provide anything else to help. Error text pasted below. Logs uploaded at the bottom.

Error: Internal error while creating updates: Message: unknown error: cannot determine loading status
from unknown error: unexpected command response
  (Session info: chrome=103.0.5060.53)
Stacktrace:
0   chromedriver                        0x000000010a6aa079 chromedriver + 4444281
1   chromedriver                        0x000000010a636403 chromedriver + 3970051
2   chromedriver                        0x000000010a2d1038 chromedriver + 409656
3   chromedriver                        0x000000010a2be3c8 chromedriver + 332744
4   chromedriver                        0x000000010a2bdac7 chromedriver + 330439
5   chromedriver                        0x000000010a2bd047 chromedriver + 327751
6   chromedriver                        0x000000010a2bbf16 chromedriver + 323350
7   chromedriver                        0x000000010a2bc44c chromedriver + 324684
8   chromedriver                        0x000000010a2c967e chromedriver + 378494
9   chromedriver                        0x000000010a2c9f22 chromedriver + 380706
10  chromedriver                        0x000000010a2d85b3 chromedriver + 439731
11  chromedriver                        0x000000010a2dc47a chromedriver + 455802
12  chromedriver                        0x000000010a2bc77e chromedriver + 325502
13  chromedriver                        0x000000010a2d81fa chromedriver + 438778
14  chromedriver                        0x000000010a33762d chromedriver + 828973
15  chromedriver                        0x000000010a324683 chromedriver + 751235
16  chromedriver                        0x000000010a2faa45 chromedriver + 580165
17  chromedriver                        0x000000010a2fba95 chromedriver + 584341
18  chromedriver                        0x000000010a67b55d chromedriver + 4253021
19  chromedriver                        0x000000010a6803a1 chromedriver + 4273057
20  chromedriver                        0x000000010a68516f chromedriver + 4292975
21  chromedriver                        0x000000010a680dea chromedriver + 4275690
22  chromedriver                        0x000000010a65a54f chromedriver + 4117839
23  chromedriver                        0x000000010a69aed8 chromedriver + 4382424
24  chromedriver                        0x000000010a69b05f chromedriver + 4382815
25  chromedriver                        0x000000010a6b18d5 chromedriver + 4475093
26  libsystem_pthread.dylib             0x00007ff805e1b4e1 _pthread_start + 125
27  libsystem_pthread.dylib             0x00007ff805e16f6b thread_start + 15

[2022-06-29_10-50-40.log](https://github.com/jprouty/mint-amazon-tagger/files/9013030/2022-06-29_10-50-40.log)

**Comment by talon256** [comment]
Failed to see this was a duplicate of #111 closing now and will comment there.
